dev-src/dev/db.clj

```diff
@@ -18,8 +18,11 @@
   (db/transact seed-tx-data))
 
 (defn reset-db! []
-  (d/delete-database (config :datomic :uri))
-  (db/initialize)
+  ;; TODO: This isn't really correct. In Datomic you shouldn't delete and
```

Comment:
It's not related to that, and I can't remember the specific behavior we saw. All I remember is that when we reported it to Cognitect they said, essentially, "Oh no no no, you shouldn't use Datomic that way." :)